File: geeksforgeeks/w2-p4-transpose-of-matrix-1587115621/Solution.js
class Solution {
    transpose(mat) {
        let n = mat.length;
        for(let i=0; i<n; i++) {
            for(let j=i+1; j<n; j++) {
                [mat[i][j], mat[j][i]] = [mat[j][i], mat[i][j]];
            }
        }
        
        return mat
    }
}

File: geeksforgeeks/w2-p4-transpose-of-matrix-1587115621/Solution.c
#include <stdio.h>

void transpose(int **mat, int **tMat, int n) {
    for(int i=0; i<n; i++) {
        for(int j=0; j<n; j++) {
            tMat[j][i] = mat[i][j];
        }
    }

    return tMat;
}
